app/src/app.py

```diff
@@ -1,7 +1,51 @@
-from flask import Flask, render_template
-
+from flask import Flask, render_template, request, redirect, flash, url_for
+from werkzeug.security import generate_password_hash
 app = Flask(__name__)
 
 @app.route('/')
 def index():
     return render_template('index.html')
+
+@app.route("/register", methods=["GET", "POST"])
+def register():
+    """Register user"""
+
+    if request.method == "POST":
+
+        #Access the data submitted to form
+        email = request.form.get("email")
+        username = request.form.get("username")
+        password = request.form.get("password")
+
```

Comment:
It may be best to add in a few more components since this is the registration page. I would suggest adding in a first and last name inputs, and password confirmation input to check if passwords match. You could also decide to put in inputs for all the database entry points that Sky added.